Tools/serial/generate_config.py

```diff
@@ -108,6 +108,13 @@
         "default_baudrate": 0,
         },
 
+    # ESP
```

Comment:
@bkueng thoughts on this part? The goal is to make the pixracer esp8266 serial port configurable. It doesn't seem to have a proper name generally documented.

Reply:
Hmm not sure what's best. The closest existing match would be TELEM4 I'd say.

Reply:
I changed it to TELEM4, however this defaults with baud rate 57600, and the esp8266 wants 921600.